view/frontend/templates/page/analytics.phtml

```diff
@@ -6,7 +6,7 @@
 $url   = $block->getLibraryUrl();
 $events = $block->getEvents();
 ?>
-<script type="text/javascript">
+<script type="text/javascript" async="true">
```

Comment:
WTF, man?! This is nonsense.

Reply:
True, actually requireJS loads all module asynchronously always.